apidoc/Titanium/Contacts/Contacts.yml

```diff
@@ -538,6 +538,7 @@ properties:
       property.
       Note: If ringtone or texttone is selected, null values are returned, since these are unsupported
       by Apple.
+      Selecting Birthday returns null in iOS 9. This is a native issue.
```

Comment:
Since iOS 9.0, there is a native apple issue whereby it may return null if the birthday property is selected on certain device models.